kubernetes/zenko/charts/cloudserver/templates/_env.tpl

```diff
@@ -111,5 +111,7 @@ env:
     value: /ssl-kmip/kmip-cert.pem
   - name: S3KMIP_CA
     value: /ssl-kmip/kmip-ca.pem
+  - name: VAULTD_HOST
+    value: "zenko-vault"
```

Comment:
I can't imagine this would always be `zenko-vault`. The name should be templated based off whatever release or naming convention being used. Also is it supposed to be within the `kmip.enabled` block?

Reply:
Great catch!
Added a block for vault. 
Created  ZENKO-1946 for making sure we template this in the future.